### PR TITLE
Convert ui reducer to slice

### LINKED ISSUE #276: Convert `ui.ts` reducer to slice

### PR DESCRIPTION
**Pull request by jefferymills**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -228,11 +229,11 @@ window.trelliscopeApp = trelliscopeApp;
 // trelliscopeApp('001a3be8', '_test/foundationtest/config.jsonp', { logger: true });
 // trelliscopeApp('fcf74975', '_test/gapminder_autocogs/config.jsonp', { logger: true });
 // trelliscopeApp('80222985', '/config.json', { logger: true, mockData: true });
-trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
+// trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
 // trelliscopeApp('62e90658', '_test/gapminder_bells/config.jsonp', { logger: true });
 // trelliscopeApp('62e90658', '_test/trelliscope-examples2/gapminder_bells/config.jsonp', { logger: true });
 // trelliscopeApp('80222985', '_test/gapminder_coggroups/config.json', { logger: true });
-// trelliscopeApp('96c61ca5', '_test/trelliscope-examples2/gapminder_reldisp/config.jsonp', { logger: true });
+trelliscopeApp('96c61ca5', '_test/trelliscope-examples2/gapminder_reldisp/config.jsonp', { logger: true });
```
Comment (dpenciso) [inline src/index.tsx]: did we want this changed?

**Review by dpenciso — CHANGES_REQUESTED** [review]

File: public/mockServiceWorker.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.47.3).
+ * Mock Service Worker (0.47.4).
```
Comment (Incin) [inline public/mockServiceWorker.js]: did we want this version incremented?

**Review by Incin — APPROVED** [review]

File: public/mockServiceWorker.js (hunk 2 above)
Comment (jefferymills) [reply public/mockServiceWorker.js]: This just happened from running `yarn install` and it's a patch update so we probably do want it.

File: src/index.tsx (hunk 1 above)
Comment (jefferymills) [reply src/index.tsx]: No, I'll fix that.

**Review by Incin — APPROVED** [review]

**Review by Incin — APPROVED** [review]

**Review by Incin — APPROVED** [review]

**Pull request merged**